packages/jest-cli/src/SearchSource.js

```diff
@@ -258,9 +274,16 @@ class SearchSource {
   getTestPaths(patternInfo: PatternInfo): Promise<SearchResult> {
     if (patternInfo.onlyChanged) {
       return this.findChangedTests({lastCommit: patternInfo.lastCommit});
+    } else if (patternInfo.findRelatedTests) {
+      return Promise.resolve(
+        this.findRelatedTestsFromPattern(patternInfo),
```

Comment:
seems like we don't need the entire patternInfo in the `findRelatedTestsFromPattern` function. Let's just pass `patternInfo.input` along? :)